CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-#Unreleased
+#v0.1.1
 - NEW ICONS: DoubleArrow, Youtube, Instagram, Twitter, Trash
```

Comment:
Fix the conflict on the changelog to have all pending changes ;)

Reply:
Done !